releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,108 @@
+.. _release-v15-0:
+
+Spring 2018 Release (v15_0)
+===========================
+
+.. ATTENTION::
+   This will be the last major release that supports Python 2. From release v16.0 (Fall 2018) on, only Python 3 will be explicitly supported. Minor (bugfix) releases of v15 will continue to support Python 2.
```

Comment:
Pretty sure “ATTENTION” doesn't really want to be block caps. There should probably also be a new line between the directive and the text it includes.